.circleci/config.yml

```diff
@@ -1,7 +1,79 @@
 version: 2.1
 orbs:
-  node: circleci/node@3.0.0
+  node: circleci/node@1.1.6
+jobs:
+  prod-build:
+    docker:
+      - image: uwbpparamedics/uw-blueprint-paramedics-react:0.0.1
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run: npm install
+      - setup_remote_docker:
+          docker_layer_caching: false
+      - run:
+          name: Install docker-compose
+          command: |
+            set -x
+            curl -L https://github.com/docker/compose/releases/download/1.12.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+            chmod +x /usr/local/bin/docker-compose
+      - run:
+          name: Run Docker compose
+          command: |
+            set -x
+            docker-compose -f docker-compose-prod.yaml up -d
+      - persist_to_workspace:
+          root: .
+          paths:
+            - .
```

Comment:
since we don't have any jobs that depend on this one, i don't think this is necessary
reference: https://circleci.com/docs/2.0/configuration-reference/#persist_to_workspace
```suggestion
```